README.md

```diff
@@ -55,6 +55,19 @@ $refund = $api->refund->fetch($refundId); // Returns a particular refund
 // Cards
 $card = $api->card->fetch($cardId); // Returns a particular card
 
+//Upi
+$upi = $api->customer->create(array('name' => 'Razorpay User', 'email' => 'customer@razorpay.com')); // Creates an customer
+$upi = $api->order->create(array('receipt' => '123', 'amount' => 100, 'currency' => 'INR','customer_id'=>'cust_4xbQrmEoA5WJ01','method'=>'upi','token'=>array('max_amount'=>20000,'expire_at'=>1634215992,'frequency'=>'monthly'),'receipt'=>'Receipt No. 1')); // Creates an order
+$upi = $api->subscription->createSubscriptionRegistration(array('customer'=>array('name'=>'Gaurav Kumar','email'=>'gaurav.kumar@example.com','contact'=>'9123456780'),'type'=>'link','amount'=>100,'currency'=>'INR','description'=>'Registration Link for Gaurav Kumar','subscription_registration'=>array('method'=>'upi','max_amount'=>'500','expire_at'=>'1634215992'),'receipt'=>'Receipt No. 5','email_notify'=>1,'sms_notify'=>1,'expire_by'=>1634215992)); // Create a Registration Link 
+$upi = $api->invoice->fetch('inv_00000000000001')->notifyBy('sms'); // Send/Resend Notifications
+$upi = $api->payment->fetch($id); // Fetch Token by Payment ID
+$upi = $api->customer->fetch($customerId)->tokens()->fetch($tokenId); // Fetch Tokens by Customer ID
```

Comment:
Actually this is to fetch the particular token by customer id. I think you should add one more to fetch all tokens by customer Id.